### ISSUE TITLE
openml/t/ID syntax problem in docker

### ISSUE THREAD

**Issue by franchuterivera** [openml/automlbenchmark#198]
Hello!

In my use case, I normally run with docker. For this reason I am interested in commands of the form:

```
python3 runbenchmark.py autosklearn openml/t/59 -m docker
```

Nevertheless, I think there is a bug that 'slugifys' the benchmark when running docker. Notice from below command output by run benchmark, that  openml/t/59 test  -> openml_t_59 test 

```
Starting docker: docker run --name autosklearn.openml_t_59.test.docker.20201127T104651 --shm-size=1024M -v /home/chico/.openml/cache:/input -v /home/chico/work/automlbenchmark_fork/results:/output -v /home/chico/.config/automlbenchmark:/custom --rm franchuterivera/autosklearn:0.11.1-stable autosklearn openml_t_59 test   -Xseed=auto -i /input -o /output -u /custom -s skip -Xrun_mode=docker .
Datasets are loaded by default from folder /home/chico/.openml/cache.
```

Is something that you guys also see or there is further things required in my original command?

**Comment by sebhrusen** [comment]
@franchuterivera please note that `master` branch is unstable, we recommend to use the `stable` branch for "daily" use.

Nevertheless, I also noticed this issue yesterday when working on https://github.com/openml/automlbenchmark/pull/196/ and included the fix there. I'll try to extract the fix and submit it soon.

**Comment by sebhrusen** [comment]
fixed with #199
